ObjectMapper/Core/Mapper.swift

```diff
@@ -412,38 +412,31 @@ extension Mapper where N: Hashable {
 }
 
 extension Dictionary {
-	internal func map<K: Hashable, V>(@noescape f: Element -> (K, V)) -> [K : V] {
```

Comment:
Take look at this blog:
http://airspeedvelocity.net/

"Your reminder that building arrays with reduce, while fun, is accidentally quadratic."